portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx

```diff
@@ -82,12 +85,30 @@ const TenantVolumes = ({
     }
   }, [loading, setErrorSnackMessage, tenantName, tenantNamespace]);
 
+  const confirmDeletePVC = (PVC: IStoragePVCs) => {
```

Comment:
```suggestion
    const confirmDeletePVC = (pvcItem: IStoragePVCs) => {
    const delPvc = {
      ...pvcItem,
      tenant:tenantName,
      namespace:tenantNamespace
    }
    setSelectedPVC(delPvc);
    setDeleteOpen(true);
  };
```

instead of mutating the incoming arguments, pass on a new object built with the help of object de structuring.